sunpy/timeseries/timeseriesbase.py

```diff
@@ -92,6 +75,11 @@ def __init_subclass__(cls, **kwargs):
 
     # kwargs are not used here but are passed in for sources.
     def __init__(self, data, meta=None, units=None, **kwargs):
+        if isinstance(data, pd.DataFrame):
+            warn_deprecated('Passing a DataFrame to GenericTimeSeries is deprecated. '
+                            'Pass to sunpy.timeseries.TimeSeries instead.')
```

Comment:
maybe I'm missing something - should this be `pass a astropy.timeseries.TimeSeries`?

Reply:
No, this is correct. I've deprecated `GenericTimeSeries(DataFrame)`, instead telling users to use the factory and do `TimeSeries(DataFrame)`.